sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/batch/TransactionalBatchOperationResult.java

```diff
@@ -0,0 +1,358 @@
+     *
+     * @return the e tag
+     */
+    public TransactionalBatchOperationResult<?> setETag(final String value) {
+        this.eTag = value;
+        return this;
+    }
+
+    /**
+     * Gets the request charge in request units for the current operation.
+     *
+     * @return Request charge in request units for the current operation.
+     */
+    public double getRequestCharge() {
+        return requestCharge;
+    }
+
+    /**
+     * Sets request charge.
+     *
+     * @param value the value
+     *
+     * @return the request charge
+     */
+    public TransactionalBatchOperationResult<?> setRequestCharge(final double value) {
+        this.requestCharge = value;
+        return this;
+    }
+
+    /**
+     * Gets the resource associated with the current result.
+     *
+     * @return Resource associated with the current result.
+     */
+    public TResource getResource() {
+        return this.resource;
+    }
+
+    /**
+     * Sets resource.
+     *
+     * @param value the value
+     *
+     * @return the resource
+     */
+    public TransactionalBatchOperationResult<TResource> setResource(final TResource value) {
+        this.resource = value;
+        return this;
+    }
+
+    /**
+     * Gets retry after.
+     *
+     * @return the retry after
+     */
+    public Duration getRetryAfter() {
+        return this.retryAfter;
+    }
+
+    /**
+     * Sets retry after.
+     *
+     * @param value the value
+     *
+     * @return the retry after
+     */
+    public TransactionalBatchOperationResult<?> setRetryAfter(final Duration value) {
+        this.retryAfter = value;
+        return this;
+    }
+
+    /**
+     * Gets status.
+     *
+     * @return the status
+     */
+    public HttpResponseStatus getStatus() {
+        return this.responseStatus;
+    }
+
+    /**
+     * Gets status code.
+     *
+     * @return the status code
+     */
+    public int getStatusCode() {
+        return this.responseStatus.code();
+    }
+
+    /**
+     * Gets sub status code.
+     *
+     * @return the sub status code
+     */
+    public int getSubStatusCode() {
+        return this.subStatusCode;
+    }
+
+    /**
+     * Sets sub status code.
+     *
+     * @param value the value
+     *
+     * @return the sub status code
+     */
+    public TransactionalBatchOperationResult<?> setSubStatusCode(final int value) {
+        this.subStatusCode = value;
+        return this;
+    }
+
+    /**
+     * Gets a value indicating whether the current operation completed successfully.
+     *
+     * @return {@code true} if the current operation completed successfully; {@code false} otherwise.
+     */
+    public boolean isSuccessStatusCode() {
+        final int statusCode = this.responseStatus.code();
+        return 200 <= statusCode && statusCode <= 299;
+    }
+
+    /**
+     * Gets response status.
+     *
+     * @return the response status
+     */
+    public HttpResponseStatus getResponseStatus() {
```

Comment:
duplicate api. you already have `getStatus()`

Reply:
Thanks. Removed!